site/content/how-to/traffic-management/response-headers.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "HTTP Response Headers"
+description: "Learn how to modify response headers for your HTTP Route using NGINX Gateway Fabric."
+weight: 700
+toc: true
+docs: ""
+---
+
+[HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/) filters can modify the headers during the request-response lifecycle. [HTTP Header Modifiers](https://gateway-api.sigs.k8s.io/guides/http-header-modifier/?h=request#http-header-modifiers) can be used to add, modify or remove headers in incoming requests. We have two types of [filter](https://gateway-api.sigs.k8s.io/api-types/httproute/#filters-optional) that can be used to instruct the Gateway for desired behaviour.
```

Comment:
I don't think we need to mention the two types of filters here since we are only showing how to use the response header filter.